_events/2016-01-31-feminism-is-for-everybody.md

```diff
@@ -0,0 +1,9 @@
+---
+identifier: feminism-is-for-everybody
+slug: feminism-is-for-everybody
+title: "January Book Club"
+date: "2017-01-31 14:00:00"
+meetup_identifier: "227648729"
```

Comment:
According to [the documentation](https://jekyllrb.com/docs/variables/) `id` is a predefined property on pages that represents "an identifier unique to the Post (useful in RSS feeds). e.g. `/2008/12/14/my-post`." For consistency (and because I come from the land of objective-c where abbreviations are forbidden) I used `identifier` everywhere.